portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -45,17 +44,17 @@ public Marker(double lat, double lng, String content) {
       this.content = content;
     }
   }
+
   private DatastoreService datastore;
 
   @Override
   public void init() {
     datastore = DatastoreServiceFactory.getDatastoreService();
   }
-  
+
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-  }
-  
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {}
```

Comment:
Is this method needed?

Reply:
I implemented it in another branch that hasn't been merged into master yet (it's waiting for billing to activate).